packages/web-app/src/sass/main.scss

```diff
@@ -4,3 +4,12 @@
 a {
   @extend .govuk-link;
 }
+
+.govuk-accordion__section-content .govuk-summary-list__row:last-child .govuk-summary-list__key, .govuk-accordion__section-content .govuk-summary-list__row:last-child .govuk-summary-list__value, .govuk-accordion__section-content .govuk-summary-list__row:last-child .govuk-summary-list__actions {
+  border-bottom: 0;
+}
+
+h2.section-heading {
+  border-top: 3px solid #1d70b8;
```

Comment:
Looks like the prototype has the border at 2px rather than 3px.

Also we could use the GOVUK variable `$govuk-brand-colour` for the blue colour rather than specifying it explicitly as this would keep consistency with the other colours.